### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse_Create new file: The error suggestion for the "Name" field is not appropriate

### ISSUE THREAD

**Issue by soumyatippapuram** [microsoft/azure-tools-for-java#4280]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse-Benchmark; #MAS3.3.3; #ErrorSuggestion; #JavatoolingforIntelliJ/Eclipse; #C+E-H; #Benchmark; #A11ySev2; 

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2020-03 (4.15.0)
Build id: 20200313-1211

**Solution Installed:** The Azure Toolkit for Eclipse 3.220, Scala IDE 4.7.x

**OS Details:**
OS: Windows (Version 1903, Build: 18362.175)

**Screen Reader:** Narrator

**MAS Violated:** MAS3.3.3

**Repro Steps:**
1.Open Java EE - Eclipse application
2.Install "Azure Toolkit for Eclipse" and "Scala Plug-ins" from Eclipse Marketplace
3.Tab till "File > New > Project" in the navigation bar and press enter
4.Tab till "HDInsight Project > Spark on HDInsight (Scala)" and select it.
5.Tab till "Next" button and press enter 
6.Enter all the required details, tab till "Next" button and press enter
7.Tab till "Module Dependencies" tab and press enter
8.Tab till "Finish" button and press enter
9.In Project Explorer, expand the created Scala project right-click src, New-->Other and press enter
10.Tab till Scala Wizards, expand and select 'Scala Object' and press next
11.Tan till the name field and without entering any name, tab away from the control
12.Check if the error message that is displayed is appropriate

**Actual Result:**
When Name is not provided, an alert message "No file path specified" is displayed. Here the error is with "Name"

**Expected Result:**
The error message should be "No name is specified"

**User impact:**
Users will not be provided with the appropriate error if the message displayed is not correct

**MAS Reference:**
[MAS 3.3.3](https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7B51EFA05C-2D41-490B-B5C0-DF220011A234%7D&file=MAS%203.3.3%20%E2%80%93%20Error%20Suggestion.docx&action=default&mobileredirect=true)

![image](https://user-images.githubusercontent.com/41108985/78984734-a1c11680-7b44-11ea-8fe3-7aec4945652f.png)

**Comment by jingyanjingyan** [comment]
This wizard is extends from original Scala Plugin. You can find the same dialog in New Project > Scala Wizards > Scala Project. So it's out of the scope of our toolkit.

**Comment by jingyanjingyan** [comment]
+ Importance of experience: Minimal. Warning message are not actually impacking users create process.
+ Quality of experience: Annoying. Users will not be provided with the appropriate error if the message displayed is not correct
+ Alternate solutions: This issue will not break users experience, no need for alternate solutions
+ Evidence: All evidence is provided in this issue.
+ [AAT Link](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076)

External issues tracked in: https://bugs.eclipse.org/bugs/show_bug.cgi?id=562991

**Comment by Alexandria-R** [comment]
Known eclipse issue, approved to close

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749652
